flyway/src/main/java/de/novatec/manufactureDbMigrator/cloud/migrator/Application.java

```diff
@@ -0,0 +1,13 @@
+package main.java.de.novatec.manufactureDbMigrator.cloud.migrator;
```

Comment:
I suspect that there's a mismatch in package setup.
Package name should go like de.novatec.tc.showcase.dbmigrator

That main.java inclusion reminds me of an misconfigured maven project folder setup